docs/src/__examples__/Drawer/DEFAULT.tsx

```diff
@@ -49,8 +49,42 @@ export default {
       </>
     );
   },
-  info: {
-    title: "Default drawer",
-    description: "Drawers should appear on a user action and be closable.",
-  },
+  exampleKnobs: [
+    {
+      component: "Dialog",
+      knobs: [
+        {
+          name: "width",
+          type: "text",
+          defaultValue: "320px",
+        },
+        {
+          name: "position",
+          type: "select",
+          options: ["right", "left"],
+          defaultValue: "right",
+        },
+        {
+          name: "title",
+          type: "text",
+          defaultValue: "",
+        },
+        {
+          name: "shown",
+          type: "boolean",
+          defaultValue: true,
+        },
```

Comment:
This `shown` doesn't appear to work, probably because this is being controlled with the button.